common/network/neighbor.h

```diff
@@ -0,0 +1,29 @@
+#ifndef __COMMON_NETWORK_NEIGHBOR_H__
+#define __COMMON_NETWORK_NEIGHBOR_H__
+
+#include <stdlib.h>
```

Comment:
Why would `<stdlib.h>` be included? It seems that the following statements do not rely on what `<stdlib.h>` declares.

Reply:
That's an oversight, thanks